projects/js-packages/components/components/pricing-slider/index.tsx

```diff
@@ -0,0 +1,53 @@
+import classNames from 'classnames';
+import React from 'react';
+import ReactSlider from 'react-slider';
+import { PricingSliderProps } from './types';
+import './style.scss';
```

Comment:
Can we please change it to use CSS module: https://github.com/Automattic/jetpack/blob/49b12cd171952f0ab65c5a2d92901b65644de073/projects/js-packages/components/components/action-button/index.jsx#L13

Reply:
react-slider` appends `-0` and `-1` to for slider colours, so it's hard to leverage CSS modules. We'll change the class names with good / long BEM class names to avoid any conflicts.